### PR TITLE
Allow building nuget package from features/readonly-ref branch

### PR DESCRIPTION
**Pull request by VSadov**
Allow building nuget package from features/readonly-ref branch

### CODE REVIEW

**Comment by VSadov** [comment]
@jaredpar for review.
FYI @dotnet/roslyn-compiler

File: src/Tools/MicroBuild/publish-assets.ps1 [hunk 1]
```diff
@@ -43,6 +43,7 @@ try
         "dev15.0.x" { } 
         "master" { } 
         "dev_pilchie_Fix389698" { }
+        "features/readonly-ref" { }
```
Comment (jcouv) [inline src/Tools/MicroBuild/publish-assets.ps1]: This won't work because of line 39 above. You need to pull more recent bits from master branch

Comment (VSadov) [inline src/Tools/MicroBuild/publish-assets.ps1]: Uh oh. Thanks!

**Comment by VSadov** [comment]
@dotnet-bot test windows_debug_vs-integration_prtest please

**Pull request merged**